bench_runner/scripts/install.py

```diff
@@ -226,6 +226,45 @@ def generate__notify(dst: Any) -> Any:
     return dst
 
 
+def generate__weekly(dst: Any) -> Any:
+    cfg = config.get_bench_runner_config()
+
+    weekly = cfg.get("weekly", {})
+    if not weekly:
+        weekly = [{"default": {"flags": [], "runners": cfg.get("runners", {}).keys()}}]
```

Comment:
This will fail because `runners`, like `weekly`, is an array of tables, so `cfg.get("runners")` gives you a list (or `None` if there are no runners _and_ the `[[runners]]` line was removed), and that list may or may not have one item that's the dict of runners.